debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2OffsetContext.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.db2;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+
+import io.debezium.connector.SnapshotRecord;
+import io.debezium.pipeline.spi.OffsetContext;
+import io.debezium.relational.TableId;
+import io.debezium.schema.DataCollectionId;
+import io.debezium.util.Collect;
+
+public class Db2OffsetContext implements OffsetContext {
+
+    private static final String SERVER_PARTITION_KEY = "server";
+    private static final String SNAPSHOT_COMPLETED_KEY = "snapshot_completed";
+    private static final String EVENT_SERIAL_NO_KEY = "event_serial_no";
```

Comment:
Is this needed for DB2? It was added for a very specific usage in SQL Server (updates, which are reflected by two rows in the CDC table). Is it the same for DB2?